internal/service/globalaccelerator/accelerator_data_source.go

```diff
@@ -44,7 +44,8 @@ func (d *acceleratorDataSource) Schema(ctx context.Context, request datasource.S
 				Computed:   true,
 			},
 			names.AttrAttributes: schema.ListAttribute{
-				Computed: true,
+				CustomType: fwtypes.NewListNestedObjectTypeOf[acceleratorAttributesModel](ctx),
+				Computed:   true,
 				ElementType: types.ObjectType{
 					AttrTypes: map[string]attr.Type{
```

Comment:
I _think_ you can use `fwtypes.AttributeTypesMust[acceleratorAttributesModel](ctx)` here to generate the `map[string]attr.Type` and not repeat everything.

Reply:
Yes, that works